layouts/webinars/single.html

```diff
@@ -98,6 +103,25 @@ <h3 class="text-orange">What you will learn:</h3>
         </div>
     </section>
 
+    <section id="gettingStartedList" class="px-4">
+        <div class="container mx-auto max-w-6xl pb-16">
+            {{ if .Params.gated }}
+                <div class="md:mx-8">
+            {{ else }}
+                <div>
+            {{ end }}
+                <h3 class="text-orange">Get Started with Pulumi:</h3>
+                <ul class="text-gray-600">
```

Comment:
You think this list will perform better than linking directly to /docs/get-started?

Reply:
I wanted the CTA to have a more natural feel on the page. And I felt this list felt like a good fit for the page over a few big buttons.